frame/support/src/dispatch.rs

```diff
@@ -1473,9 +1472,9 @@ macro_rules! decl_module {
 		$ignore:ident
 		$mod_type:ident<$trait_instance:ident $(, $instance:ident)?> $fn_name:ident $origin:ident $system:ident [ $( $param_name:ident),* ]
 	) => {
-			// We execute all dispatchable in at least one storage layer, allowing them
+			// We execute all dispatchable in a new storage layer, allowing them
 			// to return an error at any point, and undoing any storage changes.
-			$crate::storage::in_storage_layer(|| {
+			$crate::storage::with_storage_layer(|| {
 				<$mod_type<$trait_instance $(, $instance)?>>::$fn_name( $origin $(, $param_name )* ).map(Into::into).map_err(Into::into)
 			})
 	};
```

Comment:
This should not be wrapped in storage layer. This shouldn't have been there in .26 either. We had missed it. Please remove it.